GERTi/GERTiClient.lua

```diff
@@ -261,15 +382,54 @@ local function safedown()
 	end
 end
 event.listen("shutdown", safedown)
+
+-- Since we're using receivePacket *anyway*, just hook ourselves in and do an os.sleep
+-- Register event listener to receive packets from now on
+event.listen("modem_message", receivePacket)
+
+-- Wait a while to build the neighbor table.
+-- What happens here is that we get RETURNSTART messages, which we don't forward.
+-- Completely new neighbors (anything that'll appear after this is over) send AddNeighbor messages,
+--  which we then forward. It's important this is high enough for all responses to be collated.
+-- The address numbers don't actually matter at all yet
+os.sleep(5)
```

Comment:
*This code (line 395) here replaces 228 and 223.*

Reply:
Will review after changes are made.